### PR TITLE
Aura session is used for handling sessions

### PR DESCRIPTION
**Pull request by MekDrop**
Implemented [Replace session with some composer package #5](https://github.com/ImpressCMS/ideas/issues/5).

Some code parts where rewritten to use new session possibilities like namespaces.

### CODE REVIEW

File: core/providers/PreloadServiceProvider.php [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 namespace ImpressCMS\Core\Providers;
 
+use icms_preload_Handler as PreloadHandler;
```
Comment (fiammybe) [inline core/providers/PreloadServiceProvider.php]: is this related with the aura session lib?

File: core/providers/LoggerServiceProvider.php [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 namespace ImpressCMS\Core\Providers;
 
-use League\Container\ServiceProvider\AbstractServiceProvider;
 use icms_core_Logger;
-use icms_Event;
+use League\Container\ServiceProvider\AbstractServiceProvider;
 use League\Container\ServiceProvider\BootableServiceProviderInterface;
 
 /**
```
Comment (fiammybe) [inline core/providers/LoggerServiceProvider.php]: looks like this is not related to aura session, but t something else?

Comment (MekDrop) [reply core/providers/LoggerServiceProvider.php]: These lines where changed because I use auto cleanup for namespaces in PHPStorm for class definition. So, if I remove elsewhere in same class requirement for using some classes these will be automatically removed by IDE. It helps to keep code a bit cleaner.

File: core/providers/PreloadServiceProvider.php (hunk 1 above)
Comment (MekDrop) [reply core/providers/PreloadServiceProvider.php]: sometimes with session lib same preloads was invoked few times and that was it because how preloads services previously was registered in container so this change kinda related
